website/docs/internals/provider-registry-protocol.mdx

```diff
@@ -319,6 +319,9 @@ A successful result is a JSON object with the following properties:
 * `shasums_signature_url` (required): a URL from which Terraform can retrieve
   a binary, detached GPG signature for the document at `shasums_url`, signed
   by one of the keys indicated in the `signing_keys` property.
+  
+* `shasum` (required): the SHA256 checksum for this provider's zip archive as
+  recorded in the "shasums" document.
```

Comment:
Hi @sigwinch28 thank you so much for these edits! I have one more tiny correction - we try to avoid double quotes in docs. If you feel comfortable adding this in, we can get this merged for you. Thank you!

```suggestion
* `shasum` (required): the SHA256 checksum for this provider's zip archive as
  recorded in the shasums document.
```